lib/v2/uw/gix/news.js

```diff
@@ -0,0 +1,73 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const gixBaseURL = 'https://gixnetwork.org';
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category;
+
+    let newsURL = gixBaseURL + '/news';
+    let feedTitle = 'UW GIX News - ';
+    let listSelector = 'body > div.site > div.site-content > div.content-area > main.site-main > ';
+
+    switch (category) {
+        case 'blog':
+            newsURL += '/blog/';
+            feedTitle += 'Blogs';
+            listSelector += 'div.blog-wrapper > section.blog-list > article';
+            break;
+        case 'inthenews':
+            newsURL += '/inthenews/';
+            feedTitle += 'In The News';
+            listSelector += 'div.news-wrapper > section.news-list > article';
+            break;
+    }
+
+    const response = await got({
+        method: 'get',
+        url: newsURL,
+    });
```

Comment:
```suggestion
const response = await got(newsURL);
```
got request can be simplified